Commands/notify.py

```diff
@@ -0,0 +1,267 @@
+        # TODO printing of custom emojis
+        guild_data = await get_guild_data(ctx.message.guild.id)
+
+        if guild_data.notification_lists:
+            text = "Lists:\n"
+            for k, v in guild_data.notification_lists.items():
+                if v["is_custom_emoji"]:
+                    # TODO: Extract to 'get_custom_emoji' method for reusability
+                    text += ("\n<:"
+                             + ctx.bot.get_emoji(int(v["emoji"])).name
+                             + ":"
+                             + v["emoji"]
+                             + "> - "
+                             + k
+                             )
+                else:
+                    text += "\n" + v["emoji"] + " - " + k
+            # text += "\n".join(
+            #     [
+            #         v["emoji"] + " - " + k
+            #         for k, v in guild_data.notification_lists.items()
+            #     ]
+            # )
+
+            msg = await ctx.send(text)
+            for v in guild_data.notification_lists.values():
+                await msg.add_reaction(v["emoji"] if not v["is_custom_emoji"] else ctx.bot.get_emoji(int(v["emoji"])))
+
+            # TODO make reaction time configurable
+            timeout = time.time() + 60  # 1 minutes from now
+            reaction_added_task = asyncio.create_task(
+                self.wait_for_added_reactions(ctx, msg, guild_data, timeout)
+            )
+            reaction_removed_task = asyncio.create_task(
+                self.wait_for_removed_reactions(ctx, msg, guild_data, timeout)
+            )
+
+            await reaction_added_task
+            await reaction_removed_task
+            await msg.delete()
+            # TODO give option to send a message after delete, or just stay silent?
+            await ctx.channel.send("You snooze, you lose!")
+
+        else:
+            await ctx.send("No lists exist yet")
+
+    @commands.command(name="my_lists")
+    async def my_lists(self, ctx):
+        guild_data = await get_guild_data(ctx.message.guild.id)
+        subbed_lists = []
+
+        if guild_data.notification_lists:
+            for key, notification_list in guild_data.notification_lists.items():
+                if ctx.author.id in notification_list["users"]:
+                    subbed_lists.append(key)
+
+            if len(subbed_lists) > 0:
+
+                text = "Your lists are:\n - "
+                text += "\n - ".join(subbed_lists)
+                await ctx.send(text)
+            else:
+                await ctx.send("You are not subscribed to any lists.")
+        else:
+            await ctx.send("No lists exist yet")
+
+    @commands.command(name="save_config")
+    async def save_config(self, ctx):
+        await save_configs(ctx)
+        await ctx.send("Configurations saved")
+
+    @commands.command(name="add_list")
+    async def add_list(self, ctx, list_name):
+        # if not ctx.message.author.guild_permissions.administrator:
```

Comment:
Remove or uncomment :)

Reply:
This has to be uncommented, but it was necessary to comment it to test the bot because the people working on it at the time are not admins in the servers we were testing them. We did test this feature already with this uncommented and it works as well.